vgen/src/main/java/io/gapi/vgen/ruby/RubyGapicContext.java

```diff
@@ -168,7 +191,7 @@ private String returnTypeComment(Method method) {
     if (msg.hasAttribute(ElementDocumentationAttribute.KEY)) {
       String text = RDocCommentFixer.rdocify(DocumentationUtil.getScopedDescription(msg));
       text = text.trim();
-      contentBuilder.append(text.replaceAll("\\s*\\n\\s*", "\n"));
+      contentBuilder.append(text.replaceAll("[ \\t\\r\u000B]*\\n", "\n"));
```

Comment:
This actually contains two things;
- "\\n\\s*" wouldn't be great, it removes the whitespaces at the beginning of the lines (usually used for indneted codeblocks)
- "\\s" actually contains \n, therefore continuous newlines "\n\n" will be shortened to single "\n", which means removing blank lines unexpectedly.

Added comments.